changelog.d/9028.bugfix

```diff
@@ -0,0 +1 @@
+Fix a longstanding bug where some caches could grow larger than configured.
```

Comment:
Did we decide how we're spelling longstanding?

Reply:
`long-standing` is what I did in the latest release notes. 😄

Reply:
```suggestion
Fix a long-standing bug where some caches could grow larger than configured.
```